### ISSUE TITLE
On FreeBSD, signal.NSIG is smaller than biggest signal value

### ISSUE THREAD

**Issue by d1262d3a-e001-4120-aca1-c359b5bd2ab5** [python/cpython#64783]
BPO | [20584](https://bugs.python.org/issue20584)
--- | :---
Nosy | @pitrou, @vstinner, @osvenskan, @jgehrcke
Files | <li>[signal_nsig_freebsd.patch](https://bugs.python.org/file35310/signal_nsig_freebsd.patch "Uploaded as text/plain at 2014-05-22.13:05:22 by @vstinner")</li><li>[signal_nsig_freebsd-2.patch](https://bugs.python.org/file35317/signal_nsig_freebsd-2.patch "Uploaded as text/plain at 2014-05-22.20:05:12 by @vstinner")</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2014-02-10.17:36:38.170>
labels = ['type-bug', 'library']
title = 'On FreeBSD, signal.NSIG is smaller than biggest signal value'
updated_at = <Date 2014-10-25.16:16:19.959>
user = 'https://github.com/jgehrcke'
```

bugs.python.org fields:
```python
activity = <Date 2014-10-25.16:16:19.959>
actor = 'osvenskan'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Library (Lib)']
creation = <Date 2014-02-10.17:36:38.170>
creator = 'jgehrcke'
dependencies = []
files = ['35310', '35317']
hgrepos = []
issue_num = 20584
keywords = ['patch']
message_count = 10.0
messages = ['210854', '210892', '218895', '218896', '218913', '218914', '218916', '218954', '218958', '218984']
nosy_count = 6.0
nosy_names = ['pitrou', 'vstinner', 'osvenskan', 'jgehrcke', 'neologix', 'sdaoden']
pr_nums = []
priority = 'normal'
resolution = None
stage = None
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue20584'
versions = ['Python 2.7', 'Python 3.4', 'Python 3.5']
```

</p></details>

**Comment by vstinner** [comment]
https://github.com/python/cpython/issues/91145 seems to be a duplicate.

**Comment by vstinner** [comment]
I wrote https://github.com/python/cpython/pull/91929 to fix this issue and update the NSIG documentation.

**Comment by vstinner** [comment]
I marked https://github.com/python/cpython/issues/91145 as a duplicate of this issue.

**Comment by vstinner** [comment]
Fixed by commit 20cc69528677b3e5191139d1cb587531f4893b55 of https://github.com/python/cpython/pull/91929.
